app/containers/RoomItem/interfaces.ts

```diff
@@ -64,48 +64,43 @@ export interface ITypeIconProps {
 	sourceType: IOmnichannelSource;
 }
 
-export interface IRoomItemContainerProps {
-	[key: string]: string | boolean | Function | number;
-	item: any;
-	showLastMessage: boolean;
-	id: string;
-	onPress: (item: any) => void;
-	onLongPress: (item: any) => Promise<void>;
-	username: string;
-	width: number;
-	status: TUserStatus;
-	toggleFav(): void;
-	toggleRead(): void;
-	hideChannel(): void;
+interface IRoomItemTouchables {
+	toggleFav?: (rid: string, favorite: boolean) => Promise<void>;
+	toggleRead?: (rid: string, tIsRead: boolean) => Promise<void>;
+	hideChannel?: (rid: string, type: SubscriptionType) => Promise<void>;
+	onPress: (item?: any) => void;
```

Comment:
`item` is still too unclear, right?
I know it could be `TSubscriptionModel`, but this component is also used on some other places :(

Reply:
This one has a problem with types in "general" it's not just because it's used in other places, also because some types aren't so clear.